### PR TITLE
fix: profile domain tweaks

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

- [x] Use `.then()` to capture promise results
- [ ] Add i18n
- [ ] Fix Msq icon
- [ ] Update tests

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: src/domains/profile/pages/Welcome/Welcome.tsx [hunk 1]
```diff
@@ -20,7 +20,9 @@ export const Welcome = () => {
 	const [profiles, setProfiles] = useState([]);
 
 	useEffect(() => {
-		setProfiles(env.profiles().all());
+		env.profiles()
+			.all()
+			.then((profiles: any) => setProfiles(profiles));
```
Comment (faustbrian) [inline src/domains/profile/pages/Welcome/Welcome.tsx:25]: Why would a sync method have to use then? https://github.com/ArkEcosystem/platform-sdk/blob/master/packages/platform-sdk-profiles/src/repositories/profile-repository.ts#L30-L32

File: src/domains/profile/pages/Welcome/Welcome.tsx [hunk 2]
```diff
@@ -50,10 +52,10 @@ export const Welcome = () => {
 							<div className="mt-6 mb-8">
 								{profiles.map((profile: any) => (
 									<ProfileCard
-										name={profile.name()}
-										avatar={profile.avatar()}
-										balance="0"
-										key={profile.id()}
+										name={profile.name}
+										avatar={profile.avatar}
+										balance={profile.balance}
```
Comment (faustbrian) [inline src/domains/profile/pages/Welcome/Welcome.tsx:57]: This doesn't exist on the profile yet.

File: src/domains/profile/pages/Welcome/Welcome.tsx [hunk 3]
```diff
@@ -50,10 +52,10 @@ export const Welcome = () => {
 							<div className="mt-6 mb-8">
 								{profiles.map((profile: any) => (
 									<ProfileCard
-										name={profile.name()}
-										avatar={profile.avatar()}
-										balance="0"
-										key={profile.id()}
+										name={profile.name}
+										avatar={profile.avatar}
```
Comment (faustbrian) [inline src/domains/profile/pages/Welcome/Welcome.tsx:56]: `avatar` is a method.

File: src/domains/profile/pages/Welcome/Welcome.tsx [hunk 4]
```diff
@@ -50,10 +52,10 @@ export const Welcome = () => {
 							<div className="mt-6 mb-8">
 								{profiles.map((profile: any) => (
 									<ProfileCard
-										name={profile.name()}
-										avatar={profile.avatar()}
-										balance="0"
-										key={profile.id()}
+										name={profile.name}
```
Comment (faustbrian) [inline src/domains/profile/pages/Welcome/Welcome.tsx:55]: `name` is a method.

File: src/domains/profile/pages/Welcome/Welcome.tsx [hunk 5]
```diff
@@ -50,10 +52,10 @@ export const Welcome = () => {
...
+										avatar={profile.avatar}
+										balance={profile.balance}
+										key={profile.id}
```
Comment (faustbrian) [inline src/domains/profile/pages/Welcome/Welcome.tsx:58]: `id` is a method.

File: src/domains/profile/pages/Welcome/Welcome.stories.tsx [hunk 6]
```diff
@@ -7,11 +7,21 @@ export default {
...
 
-export const Default = () => (
-	<div className="w-full h-full">
-		<Welcome />
-	</div>
-);
+export const Default = () => {
+	const env = {
+		profiles: () => ({
+			all: async () => new Promise((resolve) => resolve([])),
```
Comment (faustbrian) [inline src/domains/profile/pages/Welcome/Welcome.stories.tsx:13]: This method is sync, not async. https://github.com/ArkEcosystem/platform-sdk/blob/master/packages/platform-sdk-profiles/src/repositories/profile-repository.ts#L30-L32

File: src/domains/profile/components/ProfileCard/ProfileCard.tsx [hunk 7]
```diff
@@ -28,11 +27,14 @@ export const ProfileCard = ({ avatar, name, balance, actions, onSelect, showSett
...
 				<div className="flex items-center">
-					<div
-						className="block w-12 h-12 mx-auto rounded-full sm:mx-0 sm:flex-shrink-0"
-						style={{ backgroundImage: avatar }}
-						data-testid="profile-card__user--avatar"
-					></div>
+					<div className="block w-12 h-12 mx-auto rounded-full sm:mx-0 sm:flex-shrink-0">
+						<img
+							className="rounded-full"
+							src={avatar}
```
Comment (faustbrian) [inline src/domains/profile/components/ProfileCard/ProfileCard.tsx:33]: The avatar is purely CSS, not an image.

File: src/domains/profile/pages/Welcome/Welcome.tsx (hunk 1 above)
Comment (brenopolanski) [reply src/domains/profile/pages/Welcome/Welcome.tsx:25]: Yeah are you correct. I was just looking at the story https://github.com/ArkEcosystem/desktop-wallet/blob/3.0-react/src/domains/profile/pages/Welcome/Welcome.stories.tsx#L18-L30. So for this case the promise will always log pending as long as its results are not resolved yet.

**Comment by brenopolanski** [comment]
> Pretty much all changes in this PR revert things that were done on purpose because of how the SDK works and certain is provided. What's the reasoning for that?

My mistake sorry. PR closed!

**Pull request closed without merging**